cirq/optimizers/convert_to_cz_and_single_gates_test.py

```diff
@@ -75,16 +75,16 @@ def _decompose_(self, qubits):
             yield CompositeDummy()(qubits[1])
 
     q0, q1 = cirq.LineQubit.range(2)
-    circuit = cirq.Circuit.from_ops(
+    circuit = cirq.Circuit(
         CompositeDummy()(q0),
         CompositeDummy2()(q0, q1),
     )
-    expected = cirq.Circuit.from_ops(
+    expected = cirq.Circuit(
         cirq.X(q0),
-        cirq.Y(q0) ** 0.5,
+        cirq.Y(q0)**0.5,
```

Comment:
Any idea why the formatting changed here?  I think no spaces with exponents looks slightly better, but its weird that it changed suddenly.  Did the formatter version get upgraded?

Reply:
It's just the first time these lines were hit by the autoformat. Exponentiation is done without spaces as a way of indicating it binds tighter than multiplication.